libs/packages/sam-formly/src/lib/formly-filters/sds-filters.component.ts

```diff
@@ -14,7 +14,8 @@ import * as qs from 'qs';
 @Component({
   selector: 'sds-filters',
   template: `
-      <formly-form [form]="form" [fields]="fields" [options]="options" [model]="model"></formly-form>`,
+      <formly-form [form]="form" [fields]="fields" [options]="options" [model]="model"></formly-form>
+      <button type="button" class="sds-button sds-button--primary" (click)="options.resetModel()">Reset</button>`,
```

Comment:
I think this is supposed to be "Clear All", check the design file.

Reply:
changed the button text to "Clear All"